core/bootstrap/src/main/java/org/jboss/modules/maven/MavenArtifactUtil.java

```diff
@@ -18,6 +18,15 @@
 
```

Comment:
The change from private to public is necessary, but the change in the import not. If I shouldn't change the visibility, I need to copy the two methods in the GradleResolver.